pkg/pipeline/stream/stream_test.go

```diff
@@ -323,16 +324,12 @@ func generatorSource(ctrl *gomock.Controller, logger log.CtxLogger, nodeID strin
 
 		position++
 		return record.Record{
-			// SourceID would normally be the source node ID, but since we need
-			// to add the node ID to the position to create unique positions we
```

Comment:
We don't have to create unique positions anymore, the restriction doesn't exist anymore, so this part was simplified.